models/aeif_cond_exp_implicit.nestml

```diff
@@ -115,9 +115,9 @@ neuron aeif_cond_exp_implicit:
       emit_spike()
     end
 
-    g_ex += spikeExc.get_sum()
-    g_in += spikeInh.get_sum()
-    I_stim = currents.get_sum()
+    g_ex += spikeExc * 1 nS
+    g_in += spikeInh * 1 nS
+    I_stim = currents
```

Comment:
Spike buffers are typed as real. g_ex and g_in are both nS. 
Current buffers are already typed as pA (same as I_stim in the model) so no factor is necessary.